src/main/java/com/muntian/passportinfile/services/TextFilePassportService.java

```diff
@@ -0,0 +1,42 @@
+package com.muntian.passportinfile.services;
+
+import com.muntian.passportinfile.entities.Passport;
+import com.muntian.passportinfile.entities.Visa;
+import com.muntian.passportinfile.services.accessors.PassportAccessor;
+import com.muntian.passportinfile.services.accessors.VisaAccessor;
+import com.muntian.passportinfile.services.accessors.impl.TextFilePassportAccessor;
+import com.muntian.passportinfile.services.accessors.impl.TextFileVisaAccessor;
+
+import java.util.*;
+
+//This class is my code
+public class TextFilePassportService implements PassportService {
+
+    @Override
+    public Collection<Passport> readAll(String passportFileName, String visaFileName) {
+        PassportAccessor passportAccessor = new TextFilePassportAccessor(passportFileName);
```

Comment:
File accessors should be fields and their instances should be passed using constructor.